pkg/storage/s3/s3.go

```diff
@@ -60,10 +66,41 @@ func (d *driver) CreateStorage(reqLogger logr.Logger, instance *veleroInstallCR.
 	switch {
 	// We don't yet have a bucket name selected
 	case instance.Status.StorageBucket.Name == "":
-		err = setInstanceBucketName(d, s3Client, reqLogger, instance)
```

Comment:
I think this is a rebase error when pulling in Alex's code into your branch

Reply:
Yes... good catch.